lib/dispatcher/client-h2.js

```diff
@@ -375,6 +381,7 @@ function writeH2 (client, request) {
       session[kOpenStreams] -= 1
       if (session[kOpenStreams] === 0) session.unref()
     })
+    stream.setTimeout(requestTimeout)
```

Comment:
I would not call this if there isn't a requestTimeout.

Reply:
Do you mean that if no `bodyTimeout` passed at a request level, we just ignore it, discarding the one set for the client itself? Or do you have something else in mind?